kosat-core/src/commonMain/kotlin/org/kosat/SolverTypes.kt

```diff
@@ -61,6 +53,18 @@ value class Lit(val inner: Int) {
 
     /** Is the literal [Lit.UNDEF] or [Var.posLit]/[Var.negLit] of [Var.UNDEF] */
     val isUndef: Boolean get() = inner < 0
+
+    fun toDIMACS(): Int {
```

Comment:
Consider `toDimacs`: do not capitalize all letters in acronyms in function/class names.

See: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

This is just a recommendation (style guideline, after all), and for Java, and from Google, but nevertheless: `toDimacs` (and similar) looks more clean :)

Reply:
UPD: see also https://kotlinlang.org/docs/coding-conventions.html#choose-good-names specifically for our beloved Kotlin :)